pandas/tseries/offsets.py

```diff
@@ -3002,5 +3003,3 @@ def generate_range(start=None, end=None, periods=None,
     FY5253,
     FY5253Quarter,
 ])
-
```

Comment:
why did you change this?

Reply:
Cleaning.  Seemed a bit strange that `Nano` was being added to the dict after creation.  Looks like it was historically structured that way for compat with numpy < 1.7:

https://github.com/pandas-dev/pandas/blob/94e2194b7ab2b62126d306896ac42a8c00c4f41c/pandas/tseries/offsets.py#L2017-L2019

Structure was kept the same when the `if` statement was removed, but seems fine to directly add it to the dict now.

Reply:
k cool